### PR TITLE
Task 3 review

### PR DESCRIPTION
**Pull request by jomomg**

### CODE REVIEW

File: tests/tests_models.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        self.dojo.create_new_office("Cyan")
+        self.dojo.add_fellow("Abel", "abel@dojo.com")
+        self.dojo.save_state("test_db")
+        self.model.flush()
+        self.dojo.load_state("test_db")
+        fellow = self.model.get_fellow("Abel", "abel@dojo.com")
+        office = self.model.get_office("Cyan")
+        self.assertEqual("Abel", fellow.name)
+        self.assertEqual("Cyan", office.name)
+        os.remove("test_db.db")
```
Comment (kevintumbo) [inline tests/tests_models.py]: tests are OK ...maybe add one that shows what happens when you try to load from a file that doesn't exist

Comment (mnjaggah) [inline tests/tests_models.py]: the tests look the same. You could create two instances of the dojo and save state with one, then load_state with the other to test the two functions

File: tests/tests_models.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    def test_successfully_creates_and_loads_specified_database_file(self):
+        self.model.flush()
+        self.dojo.create_new_office("Cyan")
+        self.dojo.add_fellow("Abel", "abel@dojo.com")
+        self.dojo.save_state("test_db")
+        self.model.flush()
+        self.dojo.load_state("test_db")
+        fellow = self.model.get_fellow("Abel", "abel@dojo.com")
+        office = self.model.get_office("Cyan")
+        self.assertEqual("Abel", fellow.name)
```
Comment (Gchesta) [inline tests/tests_models.py]: One of the things that i suggest you test is whether different categories of people will be loaded.

File: tests/tests_models.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        self.dojo.load_state("default")  # load data from database to lists
+
+        office = self.model.get_office("Blue")
+        living_space = self.model.get_living_space("Red")
+        fellow = self.model.get_fellow("Peter", "peter@dojo.com")
+        staff = self.model.get_staff("Paul", "paul@dojo.com")
+
+        msg = "State was not successfully saved"
+        self.assertEqual(("Blue", "Red"), (office.name, living_space.name), msg)
+        self.assertEqual(("Peter", "Paul"), (fellow.name, staff.name), msg)
```
Comment (Gchesta) [inline tests/tests_models.py]: Will rooms without occupants be saved too? Test for this

Comment (Gchesta) [inline tests/tests_models.py]: Will rooms without occupants be saved too? Test for this

**Pull request closed without merging**